feat: elevate catalog-plugin by adding labels in about section of catalog overview page (#14391)
backstage/backstage: .changeset/empty-moles-knock.md

---
'@backstage/plugin-catalog': minor
---

> Introduced labels of entity.metadata in About section of catalog overview page in catalog-plugin.

Reviewer: ```suggestion
The `AboutCard` now also displays metadata labels of each entity.
```

Author: Hi,
Thanks @Rugvip for the suggestion. I have made the necessary changes in the code and updated it.
